libs/sdk-ui-kit/src/Dialog/DialogList/DialogListItemBasic.tsx

```diff
@@ -40,7 +40,7 @@ export const DialogListItemBasic: React.VFC<IDialogListItemComponentProps> = (pr
             cx(
                 "gd-dialog-list-item-basic",
                 "s-dialog-list-item-basic",
-                `s-dialog-list-item-${stringUtils.simplifyText(id)}`,
+                `s-dialog-list-item${stringUtils.simplifyText(id)}`,
```

Comment:
imo keeping the `-` separator makes it more "clear".

Reply:
Ok I will revert this